src/main/java/edu/hawaii/its/api/service/MembershipService.java

```diff
@@ -76,23 +76,34 @@ public MembershipResults membershipResults(String currentUser, String uid) {
         // The disjoint of basis plus include and exclude: (Basis + Include) - Exclude
         List<String> groupingMembershipPaths = disjoint(parentGroupingPaths(basisIncludeExcludePaths),
                 parentGroupingPaths(excludePaths));
+        // A list of all group paths, in which the uhIdentifier is listed (including curated groupings), so we can find the intersection with curated groupings
+        List<String> groupAndCuratedGroupingsPaths = groupingsService.allGroupPaths(uid);
+        // The list of all curated groupings
+        List<String> curatedGroupingsPaths = groupingsService.curatedGroupings();
+        // Intersect the two lists
+        groupAndCuratedGroupingsPaths.retainAll(curatedGroupingsPaths);
```

Comment:
Based on Michael's comment, the variable names would make sense simplified to a single line:
```
List<String> curatedGroupingsPaths = groupingsService.allGroupPaths(uid).retainAll(groupingsService.curatedGroupings());
```